src/mpp/Dec.hpp

```diff
@@ -1172,6 +1172,21 @@ bool jump_read_key(BUF& buf, T... t)
 	return jump_find_key<PAIRS, FAMILY, PATH>(val, IS, buf, t...);
 }
 
+template<compact::Family FAMILY, size_t SUBRULE,
+	 class DST, class BUF, class = void>
+struct is_object_readable_by_value : std::false_type {
+};
```

Comment:
Looks ugly, let's fix linter or ignore it.

Reply:
Dropped this code.